### PR TITLE
[COP-81] Change navigation styling and add two new icons

### PR DESCRIPTION
**Pull request by anthonys-ebury**
Changed the navigation menu styling to use capitalisation, change focus and hover states and spacing.
Two new icons for the menu have also been added (storage and money)

**Money Icon**
image

**Storage Icon**
image

### Menu Changes
**Before**
image

**After**
image

### CODE REVIEW

**Review by mcibique — COMMENTED** [review]
We need to update visual regression tests for other stories using the navigation link:

1. layout-container--with-navigation
2. layout-menu
3. layout-navigation-link

File: src/components/ec-navigation-link/ec-navigation-link.vue [hunk 1]
```diff
@@ -113,13 +113,14 @@ export default {
...
   @apply tw-h6;
-  @apply tw-py-12 tw-px-24;
+  @apply tw-py-20 tw-px-24;
   @apply tw-flex tw-items-center;
   @apply tw-no-underline;
   @apply tw-text-gray-8;
   @apply tw-whitespace-nowrap;
 
   font-style: normal;
+  text-transform: capitalize;
```
[the remark this answers is not in the document]
Comment (anthonys-ebury) [reply src/components/ec-navigation-link/ec-navigation-link.vue]: Updated as suggested above.

**Comment by jeger-at** [comment]
@anthonys-ebury : have you seen the comment regarding integration tests from Milos? Any progress on that one?

**Comment by anthonys-ebury** [comment]
> @anthonys-ebury : have you seen the comment regarding integration tests from Milos? Any progress on that one?

COP-81 is not currently in our sprint, so wont be progressed as yet.

**Pull request closed without merging**